__tests__/p2sh/tx-proposal-new-token.test.js

```diff
@@ -0,0 +1,430 @@
+import hathorLib from '@hathor/wallet-lib';
+import TestUtils from '../test-utils';
+import { TOKEN_DATA, AUTHORITY_VALUE } from '../integration/configuration/test-constants';
+
+const walletId = 'stub_p2sh_create_tx_proposal';
+
+describe('create-token tx-proposal api', () => {
+  beforeAll(async () => {
+    global.config.multisig = TestUtils.multisigData;
+    await TestUtils.startWallet({ walletId, multisig: true });
+  });
+
+  afterAll(async () => {
+    global.config.multisig = {};
+    await TestUtils.stopWallet({ walletId });
+  });
+
+  it('should return 200 with a valid body', async () => {
+    const response = await TestUtils.request
+      .post('/wallet/p2sh/tx-proposal/create-token')
+      .send({
+        name: 'My Custom Token',
+        symbol: 'MCT',
+        amount: 1,
+      })
+      .set({ 'x-wallet-id': walletId });
+    expect(response.status).toBe(200);
+    expect(response.body.success).toBe(true);
+    expect(response.body.txHex).toBeDefined();
+    const tx = hathorLib.helpersUtils.createTxFromHex(response.body.txHex, new hathorLib.Network('testnet'));
+    expect(tx.outputs.map(o => o.decodedScript.address.base58))
+      .toEqual(expect.arrayContaining(['wbe2eJdyZVimA7nJjmBQnKYJSXmpnpMKgG']));
```

Comment:
What's this address?

Reply:
Second address of multisigAddresses. But I changed to use the programmatic reference.